plugins/shared/info-panel/index.js

```diff
@@ -279,6 +279,14 @@ class InfoPanel {
                 // highlight when scrolling to the element with the button.
                 annotate.toggleHighlight(error.$el, $trigger);
                 annotate.toggleHighlight(error.$el, $scroll);
+
+                // Add surrounding code from error.$el to the information panel
+                // The code will appear in the error description inside
+                // tota11y-info-error-description-code
+
+                //Grab parent wrapper's html and trim the returned string
+                let surroundingCode = $(error.$el).parent().html().trim();
+                $error.find(".tota11y-info-error-description-code code").text(surroundingCode);
```

Comment:
How well does this work if the parent has some very long attribute?

For example:
```
<div class="parent" style="background: url(data:image/gif;base64,R0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJ
HSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQ
KJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEA
AQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlh
HSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQ
KJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEA
AQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlh
EAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJhR0lGODlhEAAQKJHSDFKJh)">
    <img class="bad-img" src="blah.jpg">
</div>

Is the `error.$el` then completely hidden from view?